docs/src/pages/hooks/use-is-visible.mdx

```diff
@@ -0,0 +1,11 @@
+import { HooksDemos } from '@docs/demos';
+import { Layout } from '@/layout';
+import { MDX_DATA } from '@/mdx';
+
+export default Layout(MDX_DATA.useIsVisible);
+
+## Usage
+
+`use-is-visible` is a hook that returns a boolean value indicating whether the target element is visible in the viewport. It makes use of the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) to determine visibility.
```

Comment:
`use-is-visible` is a misleading name, it may mean many things not related to the viewport. Rename hook to `use-in-viewport`.